x-pack/legacy/plugins/siem/public/components/page/network/ip_overview/index.tsx

```diff
@@ -143,28 +147,26 @@ export const IpOverview = pure<IpOverviewProps>(
         onMouseEnter={() => setShowInspect(true)}
         onMouseLeave={() => setShowInspect(false)}
       >
-        {loading && (
-          <>
-            <LoadingOverlay />
-            <LoadingPanel
-              height="100%"
-              width="100%"
-              text=""
-              position="absolute"
-              zIndex={3}
-              data-test-subj="LoadingPanelLoadMoreTable"
-            />
-          </>
-        )}
         <InspectButton
           queryId={id}
           show={showInspect}
           title={i18n.INSPECT_TITLE}
           inspectIndex={0}
         />
+
         {descriptionLists.map((descriptionList, index) =>
           getDescriptionList(descriptionList, index)
         )}
+
+        {loading && (
+          <Loader
```

Comment:
maybe loader component can manage the `overlayBackground` inside of the component

Reply:
Potentially, but I think it makes the component more flexible to allow for us to supply any hex color to the `overlayBackground` prop. If we start hardcoding colors into the component, I feel like that's something we'll have to chase with maintenance if/when we need to add support for more colors in the future. Thoughts?

Reply:
It seems to be very specific since we are using darkTheme.euiPageBackgroundColor or lightTheme.euiPageBackgroundColor each time we are using this prop. I do not think this component would accept anything else for a while but I can be wrong
```
overlayBackground={
              config.darkMode ? darkTheme.euiPageBackgroundColor : lightTheme.euiPageBackgroundColor
            }
```

Reply:
Since the current requirement of importing the dark and light themes is a product of having to use Styled Components with EUI, and it is planned to move away from Styled Components at some point in the future, I'm inclined to keep it as is for now. It just makes the component more flexible. If you feel strongly about it though, we can discuss further on a Zoom. Just let me know :)